<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1442</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generador de variables aleatorias</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/dice.png</normaloff>:/icons/icons/dice.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Metodo</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget_Metodo">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Transformada inversa</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTabWidget" name="tabWidget_Metodo_Transformada">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Distribucion Uniforme</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Limite inferior</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Limite superior</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_TransformadaInversa_Uniforme_Inferior">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_TransformadaInversa_Uniforme_Superior">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>Distribucion Exponencial</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Media</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_TransformadaInversa_Exponencial_Media">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Convolucion</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTabWidget" name="tabWidget_Metodo_Convolucion">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_7">
              <attribute name="title">
               <string>Distribucion Binomial</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Probabilidad</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Convolucion_Binomial_Probabilidad">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_6">
              <attribute name="title">
               <string>Distribucion K-Erlang</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QFormLayout" name="formLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>K</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Media</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Convolucion_Erlang_Media">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="spinBox_Convolucion_Erlang_K">
                   <property name="maximum">
                    <number>9999</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_8">
              <attribute name="title">
               <string>Distribucion Normal</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QFormLayout" name="formLayout_5">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Media</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Desviacion estandar</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Convolucion_Normal_Media">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Convolucion_Normal_Desviacion">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Composicion</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QTabWidget" name="tabWidget_Metodo_Composicion">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_9">
              <attribute name="title">
               <string>Distribucion Triangular</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QFormLayout" name="formLayout_6">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Limite inferior</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Limite superior</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Moda</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Composicion_Triangular_Inferior">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Composicion_Triangular_Superior">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Composicion_Triangular_Media">
                   <property name="maximum">
                    <double>9999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_10">
              <attribute name="title">
               <string/>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="listWidget_NumerosAleatorios">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_NumerosAleatorios"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_NumerosAleatorios_Agregar">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_NumerosAleatorios_Quitar">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Quitar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_NumerosAleatorios_Borrar">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Borrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Tabla</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Distribucion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_Datos">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_BorrarTabla">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Borrar tabla</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Calcular">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
